traits/tests/test_observe.py

```diff
@@ -417,12 +422,12 @@ class Potato(HasTraits):
 
 class PotatoBag(HasTraits):
 
-    potatos = List(Instance(Potato), comparison_mode=1)
+    potatos = List(Instance(Potato), comparison_mode=ComparisonMode.identity)
```

Comment:
I _really_ want to change this to `potatoes`, but that's out of scope for this PR.

Reply:
Oops. 😂